src/omv/ports/stm32/omv_mpconfigport.h

```diff
@@ -40,4 +40,6 @@
 #define MICROPY_BOARD_FATAL_ERROR           __fatal_error
 #define MICROPY_HW_DMA_ENABLE_AUTO_TURN_OFF (0)
 
+#define MICROPY_HW_ETH_ATTRIBUTE __attribute__((aligned(16384), section(".dma_buffer")));
```

Comment:
```suggestion
#define MICROPY_HW_ETH_ATTRIBUTE __attribute__((aligned(16384), section(".dma_buffer")));
```

Now this has been renamed upstream.